src/test/java/de/poiu/apron/io/PropertyFileReaderTest.java

```diff
@@ -425,6 +426,67 @@ public void testSingleBackslashOnLine() throws IOException {
     assertThat(readEntries.get(1)).isEqualTo(new PropertyEntry("   ", "\\ ", "   ", "", "\n"));
   }
 
+  /**
+   * This test verifies bug #11: https://github.com/hupfdule/apron/issues/11
+   * @throws IOException
+   */
+  @Test
+  public void test_CrLfEndings() throws IOException {
+    // - preparation
+    final File propertyFile= this.createTestFile(""
+        + "keyA1=valueA1\n"
+        + " keyA2  =  valueA2\n"
+        + "\tkeyA3\t=\tvalue A3\t\n"
+        + "keyA4 = very long\\\r\n"
+        + "value A4 over \\\r\n"
+        + "multiple lines\r\n"
+        + "        \n"
+        + "keyB1:valueB1\n"
+        + " keyB2 : valueB2\n"
+        + "\t keyB3\t:\t value B3 \n"
+        + "keyB4 : very long\\\n"
+        + "value B4 over \\\n"
+        + "multiple lines\\\n"
+        + "\n"
+        + "keyC1 valueC1\n"
+        + "  keyC2   valueC2\n"
+        + "\t keyC3\t\tvalue C3 \n"
+        + "keyC4   very long\\\n"
+        + "value C4 over \\\n"
+        + "\t \tmultiple lines");
+
```

Comment:
One of the most important aspects of Apron is that it is (or at least should be) 100% compatible with `java.util.Properties`. 
That is actually what this PR does. The current implementation of Apron differes in the handling of an escaped `\r\n` line termination character.

To guarantee that Apron results in exactly the same key-value-pairs all tests should include a check of the parsing result of `java.util.Properties`. You can look at any other test in this class for an example.

Therefore this should be included here:

```suggestion
    // assert our assumptions about the java.util.Properties implementation
    assertThat(javaUtilProperties.size()).as("Check assumption about java.util.Properties size").isEqualTo(4);
    assertThat(javaUtilProperties.containsKey("lf")).as("Check assumption about java.util.Properties keys").isTrue();
    assertThat(javaUtilProperties.containsKey("cr")).as("Check assumption about java.util.Properties keys").isTrue();
    assertThat(javaUtilProperties.containsKey("crlf_both_escaped")).as("Check assumption about java.util.Properties keys").isTrue();
    assertThat(javaUtilProperties.containsKey("crlf_only_first_escaped")).as("Check assumption about java.util.Properties keys").isTrue();
    assertThat(javaUtilProperties.getProperty("lf")).as("Check assumption about java.util.Properties values").isEqualTo("OneTwo");
    assertThat(javaUtilProperties.getProperty("cr")).as("Check assumption about java.util.Properties values").isEqualTo("OneTwo");
    assertThat(javaUtilProperties.getProperty("crlf_both_escaped")).as("Check assumption about java.util.Properties values").isEqualTo("OneTwo");
    assertThat(javaUtilProperties.getProperty("crlf_only_first_escaped")).as("Check assumption about java.util.Properties values").isEqualTo("OneTwo");

```